### PR TITLE
accept grpc endpoint or /graphql endpoint in from_cloud function

### LINKED ISSUE #189: DgraphClientStub.from_cloud should work with the grpc endpoint
client init for cloud accept the graphql endpoint which is confusing as the client is not a GraphQL endpoint.
It should accept the gRPC endpoint (as well as graphql endpoint for backward compatibility).

### PR DESCRIPTION
**Pull request by rderbier**
This PR also deprecates from_slash_endpoint function. Fixes #189

### CODE REVIEW

File: pydgraph/client_stub.py [hunk 1]
```diff
@@ -97,12 +97,18 @@ def from_slash_endpoint(cloud_endpoint, api_key):
...
     # client = pydgraph.DgraphClient(client_stub)
+    # accept grpc endpoint as copied in cloud console
+    # accept graphql endpoint
     @staticmethod
     def from_cloud(cloud_endpoint, api_key):
         """Returns Dgraph Client stub for the Slash GraphQL endpoint"""
-        url = urlparse(cloud_endpoint)
-        url_parts = url.netloc.split(".", 1)
-        host = url_parts[0] + ".grpc." + url_parts[1]
+        if cloud_endpoint.startswith("http"): # catch http:// and https://
```
Comment (mangalaman93) [inline pydgraph/client_stub.py:105]: could we improve the logic similar to https://github.com/dgraph-io/dgo/pull/159

**Review by mangalaman93 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -482,7 +482,7 @@ There are many ways to set up your local Python environment. We suggest some san
 To build and install pydgraph locally, run
 
 ```sh
-pip install -e .[dev]
+pip install -e ".[dev]"
```
Comment (joshua-goldstein) [inline README.md:485]: why the quotes?

Comment (mangalaman93) [reply README.md:485]: zsh interprets it different and complains.

**Review by mangalaman93 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (joshua-goldstein) [reply README.md:485]: :+1:

**Review by joshua-goldstein — APPROVED** [review]

**Pull request merged**